ecosystem/sep-0012.md

```diff
@@ -140,21 +140,17 @@ returned in the response instead.
 
 Different types of customers may have different KYC requirements depending on the action a given customer wants to take.
 The `type` parameter is used to specify the action a customer wants to make so the server can identify what information
-must be collected. SEP-12 does not define what `type` values are accepted and instead leaves that to the other
-protocols, such as [SEP-6](sep-0006.md) and [SEP-31](sep-0031.md), that use SEP-12 for the actions associated with those
-protocols.
+must be collected.
```

Comment:
Let's leave the part about SEP-31 defining the types in. Otherwise, we have to define SEP-31 types here (line 152), creating coupling that we don't necessarily want.

Reply:
I agree, we should leave this part here